docs/style.md

```diff
@@ -2,45 +2,53 @@
 
 This is a set of recommended InSpec rules you should use when writing controls.
 
-## Control files
+# Control Files
 
-### 1. All controls should be located in the "controls" directory and end in ".rb"
+## Place control files in "controls/" and end them with ".rb"
 
-Reason: Most syntax highlighters will render InSpec files correctly across a wide list of tools.
+Most syntax highlighters will render InSpec files correctly across a wide list
+of tools.
 
-Avoid: `controls/ssh_config`
-Use: `controls/ssh_config.rb`
+Avoid: `controls/ssh_config`</br>
+Use: `controls/ssh_config.rb`</br>
 
-Avoid: `controls/ssh/config.rb`
+Avoid: `controls/ssh/config.rb`</br>
 Use: `controls/ssh_config.rb`
 
-### 2. Avoid using "controls" or "control" in the name of your control files
+## Avoid "controls"/"control" in your control filenames
```

Comment:
same here

Reply:
Same issue here, making this `"controls" or "controls" cause it to wrap in a bad way when rendered.